.github/dependabot.yaml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
how often will we have to update this version instead 😂

Reply:
Haha. No free lunch 🙈 

It's actually quite stable, and I haven't needed to change it in a couple of years, I think.